### PR TITLE
build: remove the warning when the version Git tag is not found

### PR DESCRIPTION
**Pull request by tomi-font**
Cherry picked from https://review.trustedfirmware.org/c/TF-M/trusted-firmware-m/+/29897.

### CODE REVIEW

**Review by ithinuel — COMMENTED** [review]
Waiting on the TF-M resolution to see if/how they want it to fix that before providing an approval on this PR. 👀

**Review by Vge0rge — APPROVED** [review]

**Review by ithinuel — APPROVED** [review]

**Pull request merged**
